network/src/main/scala/com/linkedin/norbert/network/partitioned/PartitionedNetworkClient.scala

```diff
@@ -25,7 +25,6 @@ import netty.NettyPartitionedNetworkClient
 import client.NetworkClientConfig
 import cluster.{Node, ClusterDisconnectedException, InvalidClusterException, ClusterClientComponent}
 import scala.util.Random
-import java.util
```

Comment:
While I appreciate the cleanup it would be better for unrelated changes to be in a separate pull request